main.nf

```diff
@@ -6,162 +6,195 @@
       -profile                      Configuration profile to use. Can use multiple (comma separated)
-                                    Available: standard, conda, docker, singularity, awsbatch, test
-
-    Options:
-      --singleEnd                   Specifies that the input is single end reads
-
-    References                      If not specified in the configuration file or you wish to overwrite any of the references.
-      --fasta                       Path to Fasta reference
-
+                                    Available: conda, docker, singularity, awsbatch, test and more
+
+    Alternative inputs:
+      --peptides                    Path to TSV file containing peptide sequences (minimum required: id and sequence column)
+    
+    Pipeline options:
+      --filter_self                 Specifies that peptides should be filtered against the specified human proteome references Default: false
+      --wild_type                   Specifies that wild-type sequences of mutated peptides should be predicted as well Default: false
+      --mhc_class                   Specifies whether the predictions should be done for MHC class I or class II. Default: 1
+      --peptide_length              Specifies the maximum peptide length Default: MHC class I: 8 to 11 AA, MHC class II: 15 to 16 AA 
+      --tools                       Specifies a list of tool(s) to use. Available are: 'syfpeithi', 'mhcflurry', 'mhcnuggets-class-1', 'mhcnuggets-class-2'. Can be combined in a list separated by comma.
+
+    References                      If not specified in the configuration file or you wish to overwrite any of the references
+      --reference_genome            Specifies the ensembl reference genome version (GRCh37, GRCh38) Default: GRCh37
+      --reference_proteome          Specifies the reference proteome(s) used for self-filtering
+
+    Additional inputs:
+      --protein_quantification      Path to protein quantification file (MaxQuant) for additional annotation
+      --gene_expression             Path to gene expression file for additional annotation
+      --differential_gene_expression  Path to differential gene expression file for additional annotation
+      --ligandomics_identification  Path to ligandomics identification file for additional annotation
+       
     Other options:
       --outdir                      The output directory where the results will be saved
       --email                       Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
       -name                         Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic.
+      --maxMultiqcEmailFileSize     Theshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
```

Comment:
Update this params

Reply:
Already done - it was updated everywhere but the help didn't specify this :-)